RealtimeCSG/Assets/Plugins/RealtimeCSG/Editor/Scripts/View/GUI/SceneViewBottomBarGUI/SceneViewBottomBar.GUI.cs

```diff
@@ -24,13 +24,8 @@ public static void ShowGUI(SceneView sceneView, bool haveOffset = true)
 				Rect  bottomBarRect;
 				if (haveOffset)
 				{
-#if UNITY_2021_2_OR_NEWER
-					bottomBarRect = new Rect(0, height - (CSG_GUIStyleUtility.BottomToolBarHeight + 27),
-						  width, CSG_GUIStyleUtility.BottomToolBarHeight);
-#else
 					bottomBarRect = new Rect(0, height - (CSG_GUIStyleUtility.BottomToolBarHeight + 18), 
 											  width, CSG_GUIStyleUtility.BottomToolBarHeight);
-#endif
```

Comment:
This was part of the other PR, are you sure this should've been removed?

Reply:
yes. its not needed with changes from this PR, and does basically nothing as a result